Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -3490,7 +3492,8 @@ void RenderLayer::paintLayerByApplyingTransform(GraphicsContext& context, const
     // Now do a paint with the root layer shifted to be us.
     LayerPaintingInfo transformedPaintingInfo(paintingInfo);
     transformedPaintingInfo.rootLayer = this;
-    transformedPaintingInfo.paintDirtyRect = LayoutRect(encloseRectToDevicePixels(valueOrDefault(transform.inverse()).mapRect(paintingInfo.paintDirtyRect), deviceScaleFactor));
+    if (!transformedPaintingInfo.paintDirtyRect.isInfinite())
```

Comment:
As we use InfiniteRect as input when painting pattern content right?

Reply:
Right, maybe a std::optional would have been better.  Anyway this was already a fix in the mask/clip PR from yesterday which landed.